doc/dev/decisionrecords/NamingConventions.md

```diff
@@ -62,6 +62,12 @@ them.
 | `setStop(Stop stop)`                        | Set a mutable Stop reference. Avoid if not part of natural lifecycle. Use `initStop(...)` if possible. |
 | `getStop() : Stop`                          | Old style accessor. Use the shorter form `stop() : Stop`.                                              |
 
+
+## Variables
+
+We follow the Java standard of using camelCase with a lower-case first letter for variable names. However, variables of numeric types may include units as an underscore-separated suffix. For example, `length_mm` or `duration_msec`. The reason is that case is significant in metric prefixes (e.g. `M` for mega- versus `m` for milli-). The underscore also reinforces that these are units rather than part of the base variable name.
```

Comment:
```suggestion
In general we follow the [Google Java Style Guide - Naming](https://google.github.io/styleguide/javaguide.html#s5-naming) conventions. 

We follow the standard of using camelCase with a lower-case first letter for variable names. However, variables of numeric types _may_ include units as an underscore-separated suffix. For example, `length_mm` or `duration_msec`. The reason is that case is significant in metric prefixes (e.g. `M` for mega- versus `m` for milli-). The underscore also reinforces that these are units rather than part of the base variable name. Use the full version in constants, `SLACK_IN_SECONDS`, not ~~`SLACK_S`~~.
```

Reply:
We should include the following in the first paragraph on this page:

 > We follow the [Google Java Style Guide - Naming](https://google.github.io/styleguide/javaguide.html#s5-naming) conventions. The code formatting part of the style guide is not relevant, the code is auto-formatted using prettier.

When we started using prettier we deleted the reference to the Google Java Style guide. But, prettier is only about formatting, not naming.

Reply:
I will add this reference back in and clarify. I agree, this confused me at first because the Google naming guide disappearing implied Prettier handled naming, but then I realized it only handles formatting.